code/datums/traits/good.dm

```diff
@@ -183,4 +183,68 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
-	medical_record_text = "Patient demonstrates a disturbing capacity for eating."
\ No newline at end of file
+	medical_record_text = "Patient demonstrates a disturbing capacity for eating."
+
+/datum/quirk/accent
+	name = "Accent"
+	desc = "You have developed an accent."
+	value = -1
```

Comment:
value = 0

Reply:
I have to move this out of code/datums/traits/good.dm sometime into neutral some time.
I will do this after I finish making the trait work as I want it to.